lib/cli/watch-run.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const utils = require('../utils');
+const Context = require('../context');
+const Mocha = require('../mocha');
+
+/**
+ * Exports the `watchRun` function that runs mocha in "watch" mode.
+ * @see module:lib/cli/run-helpers
+ * @module
+ * @private
+ */
+
+/**
+ * Run Mocha in "watch" mode
+ * @param {Mocha} mocha - Mocha instance
+ * @param {Object} opts - Options
+ * @param {string[]} [opts.extension] - List of extensions to watch
```

Comment:
Why not plural if it's a list?

Reply:
Because it's the canonical name of the `extension` option.
We already have a total mess in the Mocha constructor with canonical names, aliases, camelcased version ...

Reply:
Well, the function argument's name isn't the same thing as the Mocha option (though they may be one and the same in usage). Just because it's frak'd up elsewhere is no reason to propogate further into new code.